### PR TITLE
[MRG+1] changes to MEP12/sphinx-gallery compliant

### PR DESCRIPTION
**Pull request by mnegus01**
In response to #7206; similar to #8209

### CODE REVIEW

File: examples/mplot3d/pathpatch3d_demo.py [hunk 1]
```diff
@@ -1,7 +1,12 @@
...
+"""
+============================
+Draw flat objects in 3D plot
+============================
+
 Demonstrate using pathpatch_2d_to_3d to 'draw' shapes and text on a 3D plot.
-'''
+"""
 
+import numpy as np
```
Comment (NelleV) [inline examples/mplot3d/pathpatch3d_demo.py]: I wonder how this example even ran without numpy…

Comment (anntzer) [inline examples/mplot3d/pathpatch3d_demo.py]: It did not, so the question is, why was it not picked up by the gallery?

Comment (anntzer) [inline examples/mplot3d/pathpatch3d_demo.py]: http://matplotlib.org/devdocs/examples/mplot3d/pathpatch3d_demo.html super confused now.

**Comment by anntzer** [comment]
Thanks!

File: examples/mplot3d/pathpatch3d_demo.py (hunk 1 above)
Comment (NelleV) [inline examples/mplot3d/pathpatch3d_demo.py]: That is super wierd…

Comment (anntzer) [inline examples/mplot3d/pathpatch3d_demo.py]: Can you open an issue to track this?

**Comment by dstansby** [comment]
I know everyone is keen to merge doc related pull requests, but please wait until *at least* the documentation build **and** PEP8 build on travis are complete before merging.

It turns out this PR had a PEP8 violation that went by un-noticed.

**Comment by anntzer** [comment]
Apologies.

**Comment by dstansby** [comment]
Backported to `2.0.x` via. https://github.com/matplotlib/matplotlib/commit/1641eab85566af4dfbb3bbb91ae898f37163022d

**Pull request merged**